capirca/lib/paloaltofw.py

```diff
@@ -847,9 +894,22 @@ def __str__(self):
             member.text = "any"
         else:
           for x in options["source"]:
-            member = etree.SubElement(source, "member")
-            member.text = x
+            if no_addr_obj:
+              for group in address_book_groups_dict[x]:
+                member = etree.SubElement(source, "member")
+                member.text = str(address_book_names_dict[group])
+                max_src_dst += 1
+            else:
+              member = etree.SubElement(source, "member")
+              member.text = x
+              max_src_dst += 1
```

Comment:
My minor concern with this is that reusing _MAX_RULE_SRC_DST_MEMBERS here makes it seem like the limits for when the address objects are used is the same as when the address objects are not used, but that isn't the case I believe. 

I cannot recall a known limit that we discussed to modify this to, so looking for some input here. 

From looking at product docs, I see that limits are:

Address objects | 30,000 
Address groups | 15,000 
Members per address group | 2,500 <--- what we probably need?

Reply:
Oops, if address objects are used, this will be a single address group entry in the rule. Should be fine then. I have a separate concern about the address group limits that we can probably resolve outside this PR. Resolving.